plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/hms/TrinoHiveCatalog.java

```diff
@@ -297,9 +298,6 @@ public Transaction newCreateTableTransaction(
                                         .map(table -> new SchemaTableName(schema, table)))
                         .distinct())  // distinct() to avoid duplicates for case-insensitive HMS backends
                 .forEach(tablesListBuilder::add);
-
-        tablesListBuilder.addAll(listViews(session, namespace));
```

Comment:
Add `io.trino.plugin.iceberg.catalog.TrinoCatalog#listTables` javadoc

Reply:
cc @alexjo2144 for https://github.com/trinodb/trino/pull/11499

(the two changes logically conflict)